api4/plugin.go

```diff
@@ -43,7 +44,7 @@ func (api *API) InitPlugin() {
 }
 
 func uploadPlugin(c *Context, w http.ResponseWriter, r *http.Request) {
-	if !*c.App.Config().PluginSettings.Enable || !*c.App.Config().PluginSettings.EnableUploads {
+	if !*c.App.Config().PluginSettings.Enable || !*c.App.Config().PluginSettings.EnableUploads || *c.App.Config().PluginSettings.RequirePluginSignature {
```

Comment:
To confirm, do we have webapp changes that disable the button and expose this new configuration setting as well?

Reply:
We do not, yet

Reply:
No we don't yet. Created a jira for it. https://mattermost.atlassian.net/browse/MM-19844